### PR TITLE
solve scrolling problem

### LINKED ISSUE #175: X scroll on Mobile
On mobile you can scroll a little bit left and right. 
screenshot_20180315-082705

### PR DESCRIPTION
**Pull request by trabaldini**
closes #175

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -125,6 +125,11 @@
         text-align: center;
       }
 
+      .row {
+        margin-left: 0;
+        margin-right: 0;
+      }
```
Comment (marcoow) [inline index.html]: Hm, ideally we shouldn't override any of the Bootstrap styles if we can avoid it.

Comment (Turbo87) [reply index.html]: agreed. why is this needed? it sounds like maybe we're using `.row` outside of a `.container`?

File: index.html [hunk 2]
```diff
@@ -275,6 +275,9 @@
         }
         i.main-logo {
           margin-top: 160px;
+          width: 280px;
+          height: 140px;
+          background-size: 280px 140px;
```
Comment (marcoow) [inline index.html]: I think this is the wrong aspect ratio - the image is distorted on a narrow screen.

**Comment by Turbo87** [comment]
I'm assuming this was fixed by #178

**Pull request closed without merging**